vendor/github.com/quobyte/api/rpc_client.go

```diff
@@ -112,7 +113,17 @@ func (client QuobyteClient) sendRequest(method string, request interface{}, resp
 	defer resp.Body.Close()
 
 	if resp.StatusCode < 200 || resp.StatusCode > 299 {
-		log.Printf("Warning: HTTP status code for request is %s\n", strconv.Itoa(resp.StatusCode))
+		log.Printf("Warning: HTTP status code for request is %s\n",
+			strconv.Itoa(resp.StatusCode))
+		if resp.StatusCode == 401 {
+			return errors.New("Unable to authenticate with Quobyte API service")
+		}
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return (err)
+		}
+		log.Printf("Warning: Dumping full reply body:\n%s\n", string(body))
```

Comment:
We're ok with this level of detail in the logging?

Reply:
Oops, no. I was supposed to update to 0.1.8; stand by.

Reply:
Scratch that, I think we have to live with this one, because my other PR https://github.com/quobyte/api/pull/20 is in their v1.0.0 release. There is no v0.1.8.

This should be safe from the exfiltration perspective because the dial context would prevent us from getting this far.

Reply:
It's risky, we'll lose our "multiple layers of defense". Can we backport to v0.1.x?

cc @tallclair

Reply:
@venkatsc Do you think we could tag a v0.1.8 release that matches either https://github.com/quobyte/api/pull/20 or https://github.com/quobyte/api/pull/21 . My hope is that this is fine, since you have already released a v1.0.0. (Congrats, by the way!)

Reply:
Maybe we could add a strict logging flag that's enabled only in k8s and restricts logging to what you tried to do in quobyte/api#20? Quobyte might be okay with merging that change?

Reply:
Can we vendor at a commit instead of a tag?

Reply:
@misterikkit @msau42 Created a tag as requested at https://github.com/quobyte/api/tree/v0.1.8 for k8s use

Reply:
Ping @misterikkit @mattcary what's the status on this? Also @gnufied @humblec is a new heketi version cut?